packages/host/app/components/operator-mode/code-submode/playground/playground-content.gts

```diff
@@ -457,15 +480,26 @@ export default class PlaygroundContent extends Component<Signature> {
 
   private createNewCard = task(async () => {
     if (this.args.isFieldDef) {
+      let fieldCard = await loadCard(this.args.codeRef, {
```

Comment:
I apologize for the name of this function. we should really rename it so it doesn't confuse people. maybe `loadCardDef`. I'll make a ticket for that.